libbeat/_meta/config.reference.yml

```diff
@@ -620,14 +620,14 @@ output.elasticsearch:
 # Set to false to disable template loading.
 #setup.template.enabled: true
 
-# Template name. By default the template name is "beatname-%{[beat.version]}"
+# Template name. By default the template name is "beat-index-prefix-%{[beat.version]}"
 # The template name and pattern has to be set in case the elasticsearch index pattern is modified.
-#setup.template.name: "beatname-%{[beat.version]}"
+#setup.template.name: "beat-index-prefix-%{[beat.version]}"
```

Comment:
Can you have a look at line `612` and `605`. I assume we also need a change there. Sorry that I only spotted that now.

Reply:
Regarding the `beat name` at 605, could you briefly explain why you think this needs to be chnaged? From the description this still should be the name of the beat.

Regarding line 612, this basically tells you to set an index name if you want to overwrite what you have in the dashboards. There is no concrete pattern in the template, just the naming of the setting itself with `setup.dashboards.index:`.  I think this is still valid.

Reply:
For 605: You are right. I think for apm the case of multiple dashboards in one zip is even highly unlikely.

For 612: I need to check the code here if by default it does nothing or if it overwrites it with `beatname-*`. We must make sure that for apm it takes the index pattern by default or does nothing which also works.

Reply:
Here is the line that concerns me: https://github.com/elastic/beats/blob/master/libbeat/dashboards/kibana_loader.go#L62

If the dashboards are loaded through ES, empty Config.Index always seems to be ignored, but not for the Kibana API. @monicasarbu What is the default for `Config.Index` here and should it be only "overwritten" if set not empty?

Reply:
What I am saying is that 612 was empty alreay before my changes. So if there is a bug it should have nothing to do with this PR should it?

Reply:
I'm not worried about it having any affect on our beats as only there the default is changed from `apm-server-*` to `apm-*` and not consitent with the beat name.